### ISSUE TITLE
CDash test failed

### ISSUE THREAD

**Issue by JoostJM** [Radiomics/SlicerRadiomics#41]
@jcfr, it appears the generic module test for slicer radiomics failed. It is complaining about a call to VC with python arguments.

```
test_widgetRepresentation (qSlicerSlicerRadiomicsModuleGenericTest.qSlicerSlicerRadiomicsModuleGenericTest) ... Traceback (most recent call last):
  File "C:/D/N/E-0/Radiomics-build/inner-build/lib/Slicer-4.9/qt-scripted-modules/SlicerRadiomics.py", line 61, in setup
    self._addCustomizationSection()
  File "C:/D/N/E-0/Radiomics-build/inner-build/lib/Slicer-4.9/qt-scripted-modules/SlicerRadiomics.py", line 295, in _addCustomizationSection
    self.parameterFilePathLineEdit = ctk.ctkPathLineEdit(filters=ctk.ctkPathLineEdit.Files)
ValueError: Calling C++ functions with Python keywords is not supported! Function: ctkPathLineEdit(QWidget parent) Keywords: {'filters': 2}
ok
```

http://slicer.cdash.org/testDetails.php?test=8615594&build=1191518

Moreove, the [unit-tests on windows](http://slicer.cdash.org/testSummary.php?project=1&name=py_SlicerRadiomics&date=2018-02-01) also appear to be failing, but I don't see any output. The Linux/Mac tests pass, but they are tested using an earlier commit.

**Comment by JoostJM** [comment]
Update: I ran the unit test locally on my Linux build, no errors.

**Comment by JoostJM** [comment]
Update 2: The error in the generic module test appears to be due to the transition to Qt5/VTK9, I fixed this in 2cf2c9c.

**Comment by JoostJM** [comment]
@Radiomics/developers 
Some updates on the testing [cdash](http://slicer.cdash.org/index.php?project=Slicer4&date=2018-02-07&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=radiomics):
- Building with the CLI works for all platforms in slicer 4.8, still fails for Mac in nightly build (with Qt5)
- All tests are passing on Linux and Mac (with the exception of the nightly build on Mac of course)
- Windows generic module tests pass, but the SlicerRadiomicsTest [fails](http://slicer.cdash.org/index.php?project=Slicer4&date=2018-02-07&filtercount=1&showfilters=1&field1=buildname&compare1=63&value1=radiomics), it appears that it can't find the CLI... @jcfr, do you have any suggestions?
